manifests/m/Microsoft/ZunePackage/4.8.2345.0/Microsoft.ZunePackage.installer.yaml

```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Microsoft.ZunePackage
+PackageVersion: 4.8.2345.0
+InstallerSwitches:
+  Silent: /qb! /accepteula
+Installers:
+- Architecture: neutral
+  InstallerType: exe
+  InstallerUrl: https://archive.org/download/zunepackage_4.8.2345.0/ZunePackage.exe

```

Comment:
`InstallerUrl: https://archive.org/download/zunepackage_4.8.2345.0/ZunePackage.exe`

Please use an official URL for Microsoft's Zune Package.

This URL is not maintained or managed by Microsoft; and can have the executable file tampered with by anyone.